main/main.cpp

```diff
@@ -927,6 +935,18 @@ Error Main::setup(const char *execpath, int argc, char *argv[], bool p_second_ph
 			}
 		}
 
+		int mw = GLOBAL_GET("display/window/size/min_width");
+		int mh = GLOBAL_GET("display/window/size/min_height");
+		OS::get_singleton()->set_min_window_size(Size2(mw, mh));
+		
```

Comment:
There seems to be trailing whitespace here, though GitHub does not show it for some reason (normally it should be selectable).